utils/admins.go

```diff
@@ -1,6 +1,6 @@
 package utils
 
 var Admins = map[string]bool{
-	"stride1k8c2m5cn322akk5wy8lpt87dd2f4yh9azg7jlh": true, // F5
+	"stride1u20df3trc2c2zdhm8qvh2hdjx9ewh00sv6eyy8": true, // F5
```

Comment:
This address should revert to `stride1k8c2m5cn322akk5wy8lpt87dd2f4yh9azg7jlh` when `make stop-docker` is run